coderd/database/dbauthz/dbauthz.go

```diff
@@ -390,6 +390,18 @@ var (
 						policy.ActionCreate, policy.ActionDelete, policy.ActionRead, policy.ActionUpdate,
 						policy.ActionWorkspaceStart, policy.ActionWorkspaceStop,
 					},
+					// Should be able to add the prebuilds system user as a member to any organization that needs prebuilds.
+					rbac.ResourceOrganizationMember.Type: {
+						policy.ActionCreate,
+					},
```

Comment:
I think you also need read here to `determine which organizations the prebuild system user is a member of.`